### DISCUSSION TITLE
Import/export settings

### DISCUSSION THREAD

**Discussion by Dragsdal** [autobrr/autobrr#670] [Ideas]
Hello,

Thank you for a very nice program. I have used autodl-irssi so far, but have recently switched. So far: very impressed. :)

A suggestion: Make it possible to import/export settings.
I use a lot of indexers and a few download clients, and even though the GUI makes the setting up a breeze it would be helpful to be able to do an import if (in my case) eg. a wipe or flushing of the docker container is necessary.

Kind regards,
DD

**Comment by zze0s** [comment]
Hi! Happy you find it useful!

Everything is saved in a SQL database, either SQLite or Postgres so making backups and restoring is a breeze.

As for importing from autodl-irssi, [there's this issue from bp](https://github.com/autobrr/autobrr/issues/454) some time ago. It might need some updates to get fully compatible for the current builds.
